flake.nix

```diff
@@ -4,11 +4,21 @@
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
   inputs.flake-utils.url = "github:numtide/flake-utils";
 
-  outputs = { self, nixpkgs, flake-utils }:
+  outputs =
+    { self
+    , nixpkgs
+    , flake-utils
+    ,
```

Comment:
this doesn't look right, this line can be kept unchanged

Reply:
you're right, I did a mistake in the first commit. Going to fix this right now.